x-pack/plugins/cases/server/plugin.ts

```diff
@@ -109,17 +109,21 @@ export class CasePlugin {
       connectorMappingsService: this.connectorMappingsService,
       userActionService: this.userActionService,
       router,
+      subCasesEnabled: this.config.subCasesEnabled,
     });
 
-    registerConnectors({
-      actionsRegisterType: plugins.actions.registerType,
-      logger: this.log,
-      caseService: this.caseService,
-      caseConfigureService: this.caseConfigureService,
-      connectorMappingsService: this.connectorMappingsService,
-      userActionService: this.userActionService,
-      alertsService: this.alertsService,
-    });
+    if (this.config.subCasesEnabled) {
```

Comment:
We need to document that the case connector is disabled by default and add the breaking change label with release notes. Users that have created case connectors will be surprised to see that the connector cannot be used. I know that the connector is not officially supported and is not used by the UI but still I think we should at least have put a release note.

Reply:
I do not think so since we never release it this feature

Reply:
Ok!